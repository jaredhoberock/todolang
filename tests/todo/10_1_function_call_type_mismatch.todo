var x: String = "Hello";
x(1,2);
