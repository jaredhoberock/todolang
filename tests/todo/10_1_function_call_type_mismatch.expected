[0m[1m[38;5;9merror[0m[1m: Type mismatch: expected 'fn(Number, Number) -> T1', found 'String'[0m
  [0m[34mâ”Œâ”€[0m tests/todo/10_1_function_call_type_mismatch.todo:2:1
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m [0m[31mx(1,2)[0m;
  [0m[34mâ”‚[0m [0m[31m^^^^^^[0m


